sos/policies/distros/redhat.py

```diff
@@ -380,17 +411,18 @@ def upload_archive(self, archive):
         """
         try:
             if self.upload_url and self.upload_url.startswith(RH_API_HOST) and\
-              (not self.get_upload_user() or not self.get_upload_password()):
+                    (not self.get_upload_user() or
+                     not self.get_upload_password()):
                 self.upload_url = RH_SFTP_HOST
             uploaded = super(RHELPolicy, self).upload_archive(archive)
-        except Exception:
+        except Exception as e:
             uploaded = False
             if not self.upload_url.startswith(RH_API_HOST):
                 raise
             else:
                 self.ui_log.error(
-                    _(f"Upload to Red Hat Customer Portal failed. Trying "
-                      f"{RH_SFTP_HOST}")
+                    _(f"Upload to Red Hat Customer Portal failed due to "
+                      f"{e} Trying {RH_SFTP_HOST}")
```

Comment:
Missing dot at the end of sentence :) (just after `{e}`)

Reply:
Fixed